.travis.yml

```diff
@@ -38,7 +38,6 @@ matrix:
       env: LINT=1 PYTHONWARNINGS=ignore
       before_install:
         - sudo apt-get update -qq
-        - sudo apt-get install -qq clang-format-6.0
```

Comment:
The version of `clang-format` installed here is actually `clang-format-6.0`, which wasn't used. It seems that `clang-format` version `7.0.0` comes with the base image.